src/cpp/helpers/CMakeLists.txt

```diff
@@ -38,27 +38,21 @@ add_library (helpers STATIC
 		${CMAKE_CURRENT_SOURCE_DIR}/LoggerUtils.h
 		)
 
-get_target_property(JSON_INC_PATH jsoncpp_lib INTERFACE_INCLUDE_DIRECTORIES)
 get_target_property(xpansion_interfaces_path xpansion_interfaces INTERFACE_INCLUDE_DIRECTORIES)
 
 target_include_directories (helpers
 	PUBLIC
 		${CMAKE_CURRENT_SOURCE_DIR}
-		${JSON_INC_PATH}/jsoncpp
 		${xpansion_interfaces_path}
 )
 
+jsoncpplib_name(jsonlib)
 target_link_libraries (helpers 
 	PUBLIC
-		jsoncpp_lib
+		${jsonlib}
 		solvers
 		glog::glog
 		gflags::gflags
 		Boost::boost Boost::program_options
 		MINIZIP::minizip-ng
 )
-
-install(FILES
```

Comment:
i think this is need for windows installation, right?

Reply:
Indeed. I forgot a change in Toplevel CMakelist.